src/Components/Task/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { AiFillEdit, AiOutlineCheck, AiOutlineClose } from "react-icons/ai";
+import './style.css';
+
+const Task = (props) => {
+  // Flag for highlight the card
+  let [classFlag, setClassState] = useState(true);
+  const selectItem = () => setClassState(!classFlag);
+
+  // Stage for type of button and adding input fields
+  let [editFlag, setTaskState] = useState(true);
+  const changeText = () => {
+    setClassState(classFlag = true);
+    setTaskState(!editFlag);
+  };
+
+  // Intermediate storage facility for text
+  const newTextObj = { ...props.task };
```

Comment:
Why don't you put this temp data storage as a separate state entry? You're using a simple variable/constant for that purpose though this role is logically delegated to state object.

Reply:
Sorry, I didn't quite understand what you mean,
Instead of an object, I need to make a state...

Reply:
Yes, that's what I've meant. Better create a state entry for that purpose.